components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -1782,6 +1782,54 @@ public static String getIdpRoleClaimUri(ExternalIdPConfig externalIdPConfig) {
         return idpRoleClaimUri;
     }
 
+    /**
+     * Get the mapped URI for the IDP role mapping
+     * @param idpRoleClaimUri pass the IdpClaimUri created in getIdpRoleClaimUri method
+     * @param stepConfig Relevant stepConfig
+     * @param context Relevant authentication context
+     * @return idpRole claim uri in IDPs dialect or Custom dialect
+     */
+    public static String getMappedIdpRoleClaimUri(String idpRoleClaimUri, StepConfig stepConfig,
+                                                  AuthenticationContext context) {
+
+        // Set the incoming idpClaimUri
```

Comment:
This line is redundant. And also better to add a full stop at the end of the comment. Please check others.